### PR TITLE
Aj/tnl 6668 keyboard help focus

### PR DESCRIPTION
**Pull request by staubina**
### Description

Updated Drag and Drop to focus on the last selected link after closing the keyboard help

### Sandbox
- [x] Build a sandbox for your branch and add a link here
https://staubina-dnd.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/courseware/d8a6192ade314473a78242dfeedfbf5b/206ef2e06c82470a88608fd0b86893a6/

### Testing
- [x] i18n
- [x] RTL
- [x] Unit, integration, acceptance tests as appropriate

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @cahrens
- [x] Code review: @jlajoie

FYI: @sanfordstudent @sstack22 @cptvitamin

### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ install:
     - "pip install selenium==2.53.0"
     - "pip uninstall -y xblock-drag-and-drop-v2"
     - "python setup.py sdist"
-    - "pip install dist/xblock-drag-and-drop-v2-2.0.16.tar.gz"
+    - "pip install dist/xblock-drag-and-drop-v2-2.0.17.tar.gz"
```
Comment (staubina) [inline .travis.yml]: Note: This will removed before merge and a new Release PR will be created with this.

File: setup.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-drag-and-drop-v2',
-    version='2.0.16',
+    version='2.0.17',
```
Comment (staubina) [inline setup.py]: Note: This will removed before merge and a new Release PR will be created with this.

**Comment by staubina** [comment]
@cahrens, @jlajoie Please review when you have a moment.

FYI: @sanfordstudent @sstack22 @cptvitamin

**Comment by staubina** [comment]
Disregard, please wait to review.  Mark and I found a bug.

@cahrens @jlajoie

**Comment by staubina** [comment]
@cahrens @jlajoie, This PR is ready for review again.

**Comment by jlajoie** [comment]
Question, do you think there's value to have tests for checking the focus of the keyboard help element?

**Comment by cahrens** [comment]
👍 I don't think we need to add a test for this.

**Comment by cptvitamin** [comment]
👍  awesome

**Comment by jlajoie** [comment]
:+1:

**Pull request merged**